torch/export/_trace.py

```diff
@@ -436,11 +436,15 @@ def forward(self, *args, **kwargs):
                 gm, sig = aot_export(Wrapper(mod), args, **kwargs)
 
                 def strip_root(x):
-                    return (
-                        x[len("_export_root.") :]
-                        if x.startswith("_export_root.")
-                        else x
-                    )
+                    if isinstance(x, str):
+                        if x.startswith("_export_root."):
+                            return x[len("_export_root.") :]
+                        if x.startswith("_export_root"):
+                            return x[len("_export_root") :]
```

Comment:
?

Reply:
The original statement would only work `string = "export_root.var` but doesn't work for `string = "export_root"`

Reply:
then you want to use `if x == "_export_root"` to be more concise on what you're doing.